### ISSUE TITLE
cannot remove provenance information

### ISSUE THREAD

**Issue by jeanetteclark** [NCEAS/metacatui#900]
In the training, I tried to help a participant remove prov information on demo.arcticdata.io but could not. I tested myself on the same server and was able to recreate the issue - if I try to remove prov by clicking the red x on all the relationships and save, when I reload the page the prov is still there

**Comment by laurenwalker** [comment]
Was this before Chris made changes to fix the authoritative member node id in the database and MetacatUI config?

**Comment by jeanetteclark** [comment]
no, after. This package was created after Chris fixed the config and I cannot remove the prov:

https://demo.arcticdata.io/view/urn:uuid:a0a751dd-3b03-47f7-aadc-f369be4820ef

**Comment by robyngit** [comment]
In case it helps, I came across the same issue recently. I created the prov info for [this dataset](https://dev.nceas.ucsb.edu/view/urn:uuid:a843239b-6d04-4019-9835-6e3c8e3418c8) in MetacatUI. When I try to remove any part of the prov info in the editor, then save, I get the following error:

<img width="606" alt="Screen Shot 2020-12-17 at 11 22 31" src="https://user-images.githubusercontent.com/26600641/102514227-3972ef00-405a-11eb-9c8d-0c41b651f37f.png">

The saving process never finishes and I get an infinite spinner in the save button.

Potentially related issues: https://github.com/NCEAS/metacatui/issues/1517, https://github.com/NCEAS/metacatui/issues/586

**Comment by laurenwalker** [comment]
Thanks Robyn. I noticed a few months ago when preparing for a demo of the prov editor that it was VERY buggy (hence #1517, lol). I think we need to do a complete review of it.

**Comment by robyngit** [comment]
This should be fixed by the DataPackage/ResourceMap refactor (#2567). Provenance removals now modify the parsed Resource Map graph directly, and the tests verify that the relationships remain removed after serialization and reparsing.

We weren't able to confirm the original cause of this error but the legacy provenance editing and serialization logic involved in this issue has been replaced. I'm will close this as fixed by the refactor. Please reopen with the affected dataset and any browser console errors if the problem occurs again.
